### ISSUE TITLE
Change the order of labels

### ISSUE THREAD

**Issue by ItamarSoffer** [wekan/wekan#3424]
## Issue

**Placing a new label higher is not possible.**

The first labels that was created is the first for good.
I would like to add a new labels and place them high, and move old labels lower.

**Comment by basketball00011** [comment]
This would be awesome. I too find myself trying to re-order the labels.

**Comment by helioguardabaxo** [comment]
@xet7 
Out of curiosity: would an alternative be to use a function similar to initSorting in [wekan/client/components/cards/checklists.js](https://github.com/wekan/wekan/blob/b64cd358ed0af4395357423ad172b8dac9dc3178/client/components/cards/checklists.js#L9)?

**Comment by xet7** [comment]
@helioguardabaxo 

Yes, if you have time for that. Wekan uses JQuery draggable and sortable:
https://jqueryui.com/draggable/

For Wekan, search code like:
```
cd wekan
./find.sh sortable
```
Mobile web uses drag handles:
https://github.com/wekan/wekan/pulls?q=is%3Apr+is%3Aclosed+drag

**Comment by helioguardabaxo** [comment]
@xet7 
Good! I never used JQuery, but I will learn it.
Thanks!

**Comment by xet7** [comment]
@helioguardabaxo 

I don't mean you should learn JQuery. You just copy Checklists drag-drop code to be like Labels drag-drop code.
